app/assets/javascripts/ace.js.coffee

```diff
@@ -1,4 +1 @@
-#= require ace/ace
-#= require ace/mode-javascript.js
-#= require ace/mode-markdown.js
-#= require ace/mode-coffee.js
+#= require ace-rails-ap
```

Comment:
Omg! It's important.
I have a running huginn-cluster in separated docker containers deployed via Ansible.

I will try test it. How can i do it?
1) run (console or ansible task) command like this:
```
docker exec -it huginn_web bundle exec rake assets:precompile
```
2) add precompile task in docker/scripts/init
Are there any more ways?